README.md

```diff
@@ -1,132 +1,47 @@
-  replicas: 1
-  storageCapacity: 80Gi
-  storageClass: gardener.cloud-fast
-  store:
-    storageContainer: test
-    storageProvider: S3
-    storePrefix: etcd-test
-    storeSecret: etcd-backup
-  tlsClientSecret: etcd-client-tls
-  tlsServerSecret: etcd-server-tls
-status:
-  etcd:
-    apiVersion: apps/v1
-    kind: StatefulSet
-    name: etcd-test
-```
-
-## Implementation Agenda
-
-As first step implement defragmentation during maintenance windows. Subsequently, we will add zero-downtime upgrades and defragmentation.
-
-## Workflow
-
-### Deployment workflow
-
-![controller-diagram](./docs/images/controller.png)
-
-### Defragmentation workflow
-
-![defrag-diagram](./docs/images/defrag.png)
-
-## Local Setup
-
-To set up Etcd-druid locally as a pod running inside a [kind](https://kind.sigs.k8s.io/) cluster, follow this [document](docs/development/getting-started-locally.md)
+[![REUSE status](https://api.reuse.software/badge/github.com/gardener/etcd-druid)](https://api.reuse.software/info/github.com/gardener/etcd-druid) [![CI Build status](https://concourse.ci.gardener.cloud/api/v1/teams/gardener/pipelines/etcd-druid-master/jobs/master-head-update-job/badge)](https://concourse.ci.gardener.cloud/teams/gardener/pipelines/etcd-druid-master/jobs/master-head-update-job) [![Go Report Card](https://goreportcard.com/badge/github.com/gardener/etcd-druid)](https://goreportcard.com/report/github.com/gardener/etcd-druid) [![License: Apache-2.0](https://img.shields.io/badge/License-Apache--2.0-blue.svg)](LICENSE) [![Release](https://img.shields.io/github/v/release/gardener/etcd-druid.svg?style=flat)](https://github.com/gardener/etcd-druid) [![Go Reference](https://pkg.go.dev/badge/github.com/gardener/etcd-druid.svg)](https://pkg.go.dev/github.com/gardener/etcd-druid)
+
+`etcd-druid` is an [etcd](https://github.com/etcd-io/etcd) [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) which makes it easy to configure, provision, reconcile and monitor etcd clusters. It enables management of an etcd cluster through [declarative Kubernetes API model](config/crd/bases/crd-druid.gardener.cloud_etcds.yaml). 
+
+In every etcd cluster managed by `etcd-druid`, each etcd member is a two container `Pod` which consists of:
+
+- [etcd-wrapper](https://github.com/gardener/etcd-wrapper) which manages the lifecycle (validation & initialization) of an etcd.
+- [etcd-backup-restore](https://github.com/gardener/etcd-backup-restore) sidecar which currently provides the following capabilities (the list is not comprehensive):
+  - [etcd](https://github.com/etcd-io/etcd) DB validation.
+  - Scheduled [etcd](https://github.com/etcd-io/etcd) DB defragmentation.
+  - Backup - [etcd](https://github.com/etcd-io/etcd) DB snapshots are taken regularly and backed in an object store if one is configured.
+  - Restoration - In case of a DB corruption for a single-member cluster it helps in restoring from latest set of snapshots (full & delta).
```

Comment:
Should we mention about recovery from permanent quorum loss as in demo video we are showing etcd cluster with 3 member, so I'm just wondering 🤔

Reply:
Today we do not have any automatic recovery from permanent quorum loss. When we introduce tasks then we can add that.